Titanium.Web.Proxy/RequestHandler.cs

```diff
@@ -271,21 +259,17 @@ partial class ProxyServer
         }
 
 
-        private async Task<bool> HandleHttpSessionRequestInternal(TcpConnection connection,
-            SessionEventArgs args, bool closeConnection)
+        private async Task<bool> HandleHttpSessionRequestInternal(TcpConnection connection, SessionEventArgs args, bool closeConnection)
         {
+            bool dispose = true;

```

Comment:
should be disposed? I think this should be false by default and then if needs to be disposed we will set it to true. I see some inconsistency in this method.